### ISSUE TITLE
deploy-serverless fails if all resources do not have Properties keys

### ISSUE THREAD

**Issue by ndabas** [aws/aws-extensions-for-dotnet-cli#30]
When running `dotnet lambda deploy-serverless`, it fails if I have a minimal resource definition with no `Properties` key like this:

```
...
  UploadsBucket:
    Type: AWS::S3::Bucket
```

I get:

```
...
Uploading to S3. (Bucket: xxx Key: xxx.zip)
... Progress: 10%
... Progress: 21%
... Progress: 24%
... Progress: 29%
... Progress: 40%
... Progress: 51%
... Progress: 54%
... Progress: 59%
... Progress: 70%
... Progress: 81%
... Progress: 84%
... Progress: 89%
... Progress: 100%
Unknown error executing command: The given key was not present in the dictionary.
   at System.ThrowHelper.ThrowKeyNotFoundException()
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Amazon.Lambda.Tools.LambdaUtilities.UpdateCodeLocationInYamlTemplate(String templateBody, String s3Bucket, String
s3Key)
   at Amazon.Lambda.Tools.Commands.DeployServerlessCommand.<PerformActionAsync>d__63.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Amazon.Common.DotNetCli.Tools.Commands.BaseCommand`1.<ExecuteAsync>d__10.MoveNext()
```

If I just change that definition to this it works fine:

```
...
  UploadsBucket:
    Type: AWS::S3::Bucket
    Properties: {}
```

**Comment by normj** [comment]
Thanks for the bug report. I'll take a look.

**Comment by normj** [comment]
The fix for this has been shipped as part of version 3.3.1 of Amazon.Lambda.Tools 3.3.1

https://github.com/aws/aws-extensions-for-dotnet-cli/commit/eae664cedb7523eb5f90c79f49e531ed582d2380
